### PR TITLE
#961 Add getRipeConfigOptions methods

### PR DESCRIPTION
**Pull request by joaofsl**
| - | - |
| --- | --- |
| Issue |https://github.com/ripe-tech/ripe-white/issues/961|
| Decisions |Issue sparked from [this](https://github.com/ripe-tech/ripe-white/pull/1018#discussion_r901718940) conversation |

### CODE REVIEW

File: js/env.js [hunk 1]
```diff
@@ -118,3 +118,15 @@ export const getRipeWhiteAdminOptions = (environment, fallback = {}) => {
             return fallback;
     }
 };
+
+export const getRipeConfigPublicOptions = (url = null) => {
+    return {
+        url: url || "https://config-public.platforme.com/"
+    };
+};
```
[the remark this answers is not in the document]
Comment (joao-conde) [reply js/env.js]: Why not? I think we can use the same strategy where "ci" points to https://master--ripe-config.netlify.app/ and "production" to https://config.platforme.com/ at least.

File: test/env.js [hunk 2]
```diff
@@ -248,3 +248,67 @@ describe("Env", function() {
...
+        });
+        assert.deepStrictEqual(nowEnvOptions, {
+            url: "https://master--ripe-config.netlify.app/"
+        });
+        assert.deepStrictEqual(sandboxEnvOptions, {
+            url: "https://master--ripe-config.netlify.app/"
+        });
+        assert.deepStrictEqual(stageEnvOptions, {
+            url: "https://master--ripe-config.netlify.app/"
+        });
```
Comment (joao-conde) [inline test/env.js:273]: Even though they are all the same (for now) keep the same approach as the other tests suites of one test per environment.

Also add a test for the fallback.

File: test/env.js [hunk 3]
```diff
@@ -248,3 +248,67 @@ describe("Env", function() {
...
+    it("should return production option for 'production' environment", function() {
+        const productionEnvOptions = ripeCommons.getRipeConfigOptions("production");
+        assert.deepStrictEqual(productionEnvOptions, {
+            url: "https://config.platforme.com/"
+        });
+    });
+});
+
+describe("#getRipeConfigPublicOptions()", function() {
+    it("should return production option for 'ci', 'test', 'now', 'sandbox', and 'stage' environments", function() {
```
Comment (joao-conde) [inline test/env.js:285]: Same as above

**Review by joao-conde — CHANGES_REQUESTED** [review]

File: test/env.js (hunk 2 above)
Comment (joaofsl) [reply test/env.js:273]: We have multiple per environment https://github.com/ripe-tech/ripe-commons-js/blob/d639b25ed15b83d604e9bc7711ee1bf311286761/test/env.js#L7-L17
Pretty much one for every environment we have

Comment (joao-conde) [reply test/env.js:273]: You are right, missed that, thx!

Comment (joao-conde) [reply test/env.js:273]: Add the fallback tests tho

**Review by joao-conde — APPROVED** [review]

**Pull request merged**